packages/mint-components/src/components/sqm-divided-layout/sqm-divided-layout-view.tsx

```diff
@@ -35,7 +35,7 @@ export function DividedLayoutView(
     display: flex;
     flex: 1;
     flex-direction: ${props.direction};
-    background-color: var(--st-portal-background);
+    background-color: var(--st-content-background);
```

Comment:
This is `mint-components`, should we be using `st-*` variables here?

Reply:
replaced with -sqm

Reply:
When we were doing service titan some service titan styles ended up leaking into mint components. We should probably update the service titan project to have these styles and remove them from mint.

Reply:
Going to deprecate ST support in the current mint components version. I'll add the breaking changes to the changelog. Should be an easy fix on the ST end if we ever want to update them.